package.json

```diff
@@ -1,13 +1,16 @@
 {
   "name": "@github/custom-element-element",
   "version": "0.0.1",
+  "private": true,
```

Comment:
I understand we probably don't want to publish this module, but this can easily trip people up when using this boilerplate and not knowing to remove this.

Reply:
The npm error message states what to do to fix the issue but yeah this is probably unnecessary and an extra step when people are using this template.

```
npm ERR! code EPRIVATE
npm ERR! This package has been marked as private
npm ERR! Remove the 'private' field from the package.json to publish it.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/kristjanoddsson/.npm/_logs/2019-08-19T07_57_46_754Z-debug.log
```